src/SerilogTracing/TracingLoggerEnrichmentConfigurationExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Serilog;
+using Serilog.Configuration;
+using SerilogTracing.Enrichers;
+
+namespace SerilogTracing;
+
+/// <summary>
+/// Enrichment for spans.
+/// </summary>
+public static class TracingLoggerEnrichmentConfigurationExtensions
+{
+    /// <summary>
+    /// Enrich log events with the span duration as milliseconds.
+    /// </summary>
+    /// <param name="enrichment">The configuration object.</param>
+    /// <param name="propertyName">The name of the property to add.</param>
+    public static LoggerConfiguration WithElapsedMilliseconds(this LoggerEnrichmentConfiguration enrichment, string? propertyName = null)
+    {
+        return enrichment.With(new ElapsedMilliseconds(propertyName ?? "Elapsed"));
+    }
+    
+    /// <summary>
+    /// Enrich log events with the span duration as a <see cref="TimeSpan"/>.
+    /// </summary>
+    /// <param name="enrichment">The configuration object.</param>
+    /// <param name="propertyName">The name of the property to add.</param>
+    public static LoggerConfiguration WithElapsedTime(this LoggerEnrichmentConfiguration enrichment, string? propertyName = null)
```

Comment:
When these appear standalone in a `LoggerConfiguration()` method chain, it might be hard for a reader to get a handle on what "elapsed" means in that context (among all of the other config).

What about if we used `WithSpanTiming()` and `WithSpanTimingMilliseconds()` - and just bite the bullet that the property name by default won't be directly related? (It's already slightly misaligned anyway, because of the suffixes on the two method names..)